docs/en/ingest-management/elastic-agent/uninstall-elastic-agent.asciidoc

```diff
@@ -51,10 +51,10 @@ files are installed? See <<installation-layout>>.
 installed for endpoint protection. The directory structure is similar to {agent},
 for example, `/Library/Elastic/Endpoint/*`.
 +
-//TODO: Add to docs when this is confirmed.
-//NOTE: When you remove the Elastic Endpoint integration from a macOS host, the
-//Endpoint System Extension is left on disk intentionally. If you want to remove
-//the extension, refer to the documentation for your operating sytem.
+NOTE: When you remove the Elastic Endpoint integration from a macOS host
```

Comment:
```suggestion
NOTE: When you remove the {elastic-endpoint} integration from a macOS host
```

Reply:
thanks, I noticed the security docs weren't using the attribute, so I wondered if I should. Probably doesn't hurt, tho!